ts/Series/Networkgraph/NetworkgraphSeries.ts

```diff
@@ -314,6 +406,12 @@ class NetworkgraphSeries extends Series {
             });
         });
 
+        addEvent(this, 'afterSimulation', function (): void {
+            this.deferDataLabels = false;
+            this.drawDataLabels(); // first time initialization with opacity = 0
+            this.drawDataLabels(); // second time for setting the opacity = 1
```

Comment:
I don't think you need both, could you double-check it? I mean networkgraph has one group for links and nodes, so `drawDataLabels` is called "twice" anyway

Reply:
The problem is this:
- when the layout is stable the `drawDataLabels` method is not called each frame
- if the `animation.defer > simulationTime` then the `drawDataLabels` will never be called (that's why we force it here)
- the first call of `drawDataLabels` (therefore also the first call of `initDataLabels`) initializes the SVG group with an `opacity = 0`
- if we leave just one call of `drawDataLabels` in the `afterSimulation` event then they will always have `opacity = 0`
- during the second call of `initDataLabels` we see that the `dataLabels` have already been initialized so we set the `opacity = 1` and the animation starts

If I leave only one call of `drawDataLabels` the labels will always have opacity of `0`. That's why I had to add the second call.

However, I think now that instead of calling the `drawDataLabels` for the second time we could just try to set opacity of the SVG group with `this.dataLabelsGroup.attr({ opacity: 1 })` and that should work.

What do you think?

Reply:
Yes, I saw the comments.
What I wanted to say is `drawDataLabels` in Networkgraph calls `Series.drawDataLabels` twice (at this moment lines: 264 and 272 in `NetworkgraphSeries.ts`), which calls `initDataLabels` twice, thus no need to call it twice here.